.github/workflows/pipeline.yml

```diff
@@ -37,7 +66,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.x']
+        python-version: ['3.7', '3.11']
```

Comment:
```suggestion
        python-version: ['3.8', '3.11']
```

Maybe?

Reply:
I might save this for a "drop 3.7" PR in the future, if that's okay.

Reply:
That works! 👍